src/pages/WorldEditor/WorldEditorStartPanel.scss

```diff
@@ -7,6 +7,10 @@
     grid-template-columns: repeat(3, 1fr);
   }
 
+  &__sidebar {
+    height: calc(100% - #{$admin-ng-sidebar-footer-height--normal});
+  }
+
```

Comment:
There will be no need for re-calculating the Admin heights and breaking unified admin UI/UX (especially since almost all the panels now do have a general nav bar, an admin nav bar a side bar etc...).

Simply disabling the tabs from clicking should work without the extra effort and inconsistencies.